src/robot.coffee

```diff
@@ -68,6 +68,20 @@ class Robot
       @emit 'error', err
     process.on 'uncaughtException', @onUncaughtException
 
+  # Public: Adds a custom Listener with the provided matcher, options, and
```

Comment:
By making this public, it does expose Listener.matcher as a new API. The API seems fairly well defined, but I'm not certain there are explicit test cases protecting it...